macros/light-sdk-macros/src/traits.rs

```diff
@@ -377,11 +362,11 @@ mod tests {
                 pub my_program: Program<'info, MyProgram>, // Missing #[self_program]
                 pub payer: Signer<'info>, // Missing #[fee_payer]
                 pub user: AccountInfo<'info>, // Missing #[authority]
-                pub light_system_program: Program<'info, LightSystemProgram>,
-                pub registered_program_pda: Account<'info, RegisteredProgram>,
+                pub light_system_program: AccountInfo<'info>,
```

Comment:
I think we need to check this account since this is the program id we cpi. It could be a manual check in the verify method to avoid the program dependency.

Reply:
Other compression specific accounts don't need to be checked because we check them inside of the system program invocation.

Reply:
done da7be0d96c8a70788c3a4232aff4e43ecb06b554